waku/api.go

```diff
@@ -70,6 +70,8 @@ type Info struct {
 	MaxMessageSize uint32  `json:"maxMessageSize"` // Maximum accepted message size
 }
 
+// TODO does anyone know why all the PublicWakuAPI methods have an unused context.Context?
+//  Doesn't seem to be an interface implementation hangover.
```

Comment:
Does anyone know why all the `PublicWakuAPI` methods have an unused context.Context? Doesn't seem to be an interface implementation hangover.

Reply:
not sure, we expose them through RPC, so maybe that's the reason? if not needed we can remove/try without

Reply:
Yeah, i don't really see the point of the `ctx` argument. It doesn't seem to be used at all.
Just look at this:
https://github.com/status-im/status-go/blob/aa7f5915873b8cd7a0a06662fa80dbf93aeec32f/services/incentivisation/service.go#L288
https://github.com/status-im/status-go/blob/aa7f5915873b8cd7a0a06662fa80dbf93aeec32f/services/incentivisation/service.go#L295

Reply:
@cammellos and @jakubgs This has too many dependencies outside of this repo to make this worth doing. I'm reverting my initial commit that removes the unused contexts. It seems that the bindings make use of the context, even though we don't consume the context.